battery-plus

```diff
@@ -751,7 +756,12 @@ case "$status" in
         fi
         long_segments+=( "$(display_batt_fully_charged)" )
         ;;
-    pending-charge)     long_segments+=( "$(display_batt_pending_charge)" ) ;;
+    pending-charge)
+        if $_HIDE_IF_CHARGED && [ $battery_percentage -ge $_HIDE_OVER_PERCENTAGE ]; then
```

Comment:
Maybe _HIDE_OVER_PERCENTAGE should default to 100, otherwise anyone using -H is automatically opted-in to hide output if it's 85%